data/human/culture conversations.txt

```diff
@@ -80,6 +80,20 @@ outfit "Mug"
 
 
 
+
+mission "New China Mourners"
+	minor
+	source "New China"
+	to offer
+		random < 1
+	on offer
+		conversation
+			`The skies pour down onto the world below, weeping together with the crowds as a plain but stately coffin is carried slowly from a void deck. Rhythmic wailing blends with the susurrant rain, filling the air as the coffin is followed closely by procession of mourners that look to be an extensive gathering of family. Even as it is placed into the van they do not leave the departed, a number of their hands placed on it as if pushing the person towards their final resting place as the two foremost mourners, a woman and a young man, climb in and take their seats next to it. Somewhere nearby, music begins to play as the vehicle slowly moves off.`
```

Comment:
@TheMarksman-ES [your Singapore is showing](https://en.wikipedia.org/wiki/Void_deck) 😛 .  I had no clue what a void deck was until I Googled it, and I doubt I'll be the only one. Any replacement wording you can think of so the rest of us know what you're talking about without having to look it up?
```suggestion
			`The skies pour down onto the world below, weeping together with the crowds as a plain but stately coffin is carried slowly from a void deck. Rhythmic wailing blends with the susurrant rain, filling the air as the coffin is followed closely by procession of mourners that look to be an extensive gathering of family. Even as it is placed into the van they do not leave the departed, a number of their hands placed on it as if pushing the person toward their final resting place. The two foremost mourners, a woman and a young man, climb in and take their seats next to it. Somewhere nearby, music begins to play as the vehicle slowly moves off.`
```
I do really like the writing here though.

Reply:
@W1zrad Void Decks are extremely specific, so I'm not too sure there's any suitable replacement for them. That, and with my current intent to use a lot of void decks to show off some more cultural convo means it is going to get a lot more head scratching down the line.
That is, if someone finds a suitable replacement.

Reply:
I think the broader adoption of void decks on heavily populated worlds (particularly those of clear Asian influence) makes a lot of sense really.